static/src/stylesheets/module/tailor/_tailor-survey-overlay.scss

```diff
@@ -0,0 +1,128 @@
+.tailor-survey-overlay,
+.tailor-survey-background {
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+	z-index: 1050;
+	position: fixed;
+	display: block;
+}
+
+.tailor-survey-overlay .tailor-survey-background {
```

Comment:
You probably want to use BEM syntax here `tailor-survey__overlay` to define this as a sub-element of your `tailor-survey` block